README.md

```diff
@@ -1 +1,317 @@
+| inviteMessage  | string                                    | yes      | none    | customized invitation message sent to guests                                                                    |
+| dashBoardPhoto | string                                    | no       | none    | photo to display on couple's dashboard                                                                          |
+| bannerPhoto    | string                                    | no       | none    | main photo to display on invitations                                                                            |
+| galleryPhotos  | object { [number]: [url_as_string] }      | no       | none    | photos to display in carousel on invitations; keys are photo numbers and values are public URLs as strings      |
+| colors         | object { [number]: [hex_code_as_string] } | no       | none    | custom invitation color scheme picked by the couple; keys are color numbers and values are hex codes as strings |
+
+#### Sample response
+
+```json
+{
+  "eventId": 14
+}
+```
+
+---
+
+### `GET` /events/:eventId
+
+Returns an event
+
+#### Path parameters
+
+| Parameter | Type    | Required | Default | Description                        |
+| :-------- | :------ | :------- | :------ | :--------------------------------- |
+| eventId   | integer | yes      | none    | id of the event you are retrieving |
+
+#### Sample response
+
+```json
+{
+  "eventId": 12,
+  "coupleName1": "Ted Baker",
+  "coupleName2": "Yves St-Laurent",
+  "email": "couple@email.com",
+  "date": "2022-08-12T20:17:46.384Z",
+  "venue": "Hack Reactor Auditorium",
+  "addressLine1": "119 Nueces St",
+  "addressLine2": "Room 1104",
+  "city": "Austin",
+  "state": "TX",
+  "zipCode": "78701",
+  "daysToEvent": 15,
+  "guestLimit": 100,
+  "rsvpDeadline": "2021-12-01T20:17:46.384Z",
+  "daysToRSVPDeadline": 12,
+  "inviteMessage": "Please join us for our special day",
+  "dashboardPhoto": "shorturl.at/oJW34",
+  "bannerPhoto": "shorturl.at/oJW34",
+  "galleryPhotos": {
+    "1": "shorturl.at/oJW34",
+    "2": "shorturl.at/oJW34",
+    "3": "shorturl.at/oJW34"
+  },
+  "colors": {
+    "1": "#F0DA32",
+    "2": "#87B3BF",
+    "3": "#6E4AFF",
+    "4": "#5337C9"
+  },
+  "guests": {
+    "johnson family": [
+      {
+        "guestId": 12,
+        "firstName": "John",
+        "lastName": "Doe",
+        "email": "local@domain.com",
+        "rsvpStatus": "attending",
+        "rvspLastUpdated": "2021-05-23T20:17:46.384Z",
+        "rsvpNote": "I look forward to attending!",
+        "group": "joel family"
+      },
+      {
+        "guestId": 13,
+        "firstName": "Jane",
+        "lastName": "Doe",
+        "email": "local@domain.com",
+        "rsvpStatus": "not attending",
+        "rvspLastUpdated": "2021-05-13T20:17:46.384Z",
+        "rsvpNote": "I will not be attending your wedding!",
+        "group": "johnson family"
+      }
+    ],
+    "family friends": [
```

Comment:
Would the guests be split up into respective groups and guest objects wouldn't have a group property? Either works, but I noticed an inconsistency among guests in "johnson family" and those in "family friends"

Reply:
See above